scripts/spotless.gradle

```diff
@@ -0,0 +1,18 @@
+apply plugin: "com.diffplug.spotless"
+
+spotless {
+    encoding 'UTF-8'
+    kotlin {
+        target "**/*.kt"
+        ktlint()
```

Comment:
I would prefer to follow the default one (They will be updating it periodically) and then we don't need to take care of it.
On the repository, we have a [plugin](https://github.com/ben-manes/gradle-versions-plugin) that notifies us of new versions of the dependencies/plugin we have on the app, but it is not able to check the KtLint version used in this plugin, that is because I didn't specify it.

Reply:
That makes sense!